### DISCUSSION TITLE
No option for Colombia

### DISCUSSION THREAD

**Discussion by zofbtb** [community/community#37915] [Other Feature Feedback, Questions, & Ideas]
Hi,

I've got your 6.3.6 version on a pre Matrix Kodi box. It works great with PIA VPN but (unlike their Android app) there's no entry for Colombia.

Could this be added?

Many thanks

**Comment by ossanna16** [comment]
Hi there @zofbtb and welcome to our community! Can you please elaborate on your question so we can try to help?

**Comment by zofbtb** [comment]
just zap it - hit the wrong button whilst using a crappy mobile phone!

**Reply by ossanna16** [reply]
Thanks for letting us know!
